src/library_html5.js

```diff
@@ -2205,7 +2205,9 @@ var LibraryJSEvents = {
 
   _emscripten_webgl_power_preferences: "['default', 'low-power', 'high-performance']",
 
-#if !USE_PTHREADS
+// In offscreen framebuffer mode, we implement these functions in C so that they enable
+// the proxying of GL commands. Otherwise, they are implemented here in JS.
+#if !(USE_PTHREADS && OFFSCREEN_FRAMEBUFFER)
```

Comment:
@kainino0x mentioned in a separate email thread that some apps might be compiling with both OFFSCREENCANVAS_SUPPORT=1 and OFFSCREEN_FRAMEBUFFER=1. The intent would be that offscreen canvas support with non-proxied WebGL would be preferred, but could fall back to proxying.

It's OK to move forward if that mode isn't currently working as expected - we can refine this, and it's best to get OFFSCREENCANVAS_SUPPORT working in its intended form sooner rather than later.